cleanlab/dataset.py

```diff
@@ -305,13 +309,14 @@ def overall_label_health_score(
         A score between 0 and 1, where 1 implies all labels in the dataset are estimated to be correct.
         A score of 0.5 implies that half of the dataset's labels are estimated to have issues.
     """
+    if multi_label:
+        raise ValueError("overall_label_health_score is not supported when multi_label is True.")
```

Comment:
TODO: can you make this method work for the multi-label setting, in a follow-up PR after we've finalized this one.

Let me know if you have questions then, but let's first fully figure out the `common_multilabel_issues` method

Reply:
Added: https://github.com/cleanlab/cleanlab/blob/361e05284c5bb5151a49aee7e5aa8ff41ad47220/cleanlab/multilabel_classification/dataset.py#L200